### PR TITLE
render: switch to generic command from library-go

### PR DESCRIPTION
**Pull request by sttts**

### CODE REVIEW

**Comment by sjenning** [comment]
after a brief review
/lgtm

**Comment by aveshagarwal** [comment]
would prefer to get https://github.com/openshift/cluster-kube-scheduler-operator/pull/22 in first.

**Comment by sttts** [comment]
> would prefer to get #22 in first.

Noticed #22 too late. Feel free to take over my branch.

**Comment by aveshagarwal** [comment]
No idea why "/hold cancel" is not working and it seems CI is eating/hiding "/hold cancel" comments

File: cmd/cluster-kube-scheduler-operator/render/render.go [hunk 1]
```diff
@@ -1,53 +1,40 @@
...
 func NewRenderCommand() *cobra.Command {
-	renderOpts := &renderOpts{}
+	renderOpts := renderOpts{
+		generic:  *genericrenderoptions.NewGenericOptions(),
+		manifest: *genericrenderoptions.NewManifestOptions("kube-scheduler", "openshift/origin-hyperkube:latest"),
+	}
 	cmd := &cobra.Command{
 		Use:   "render",
-		Short: "Render kube-scheduler bootstrap manifests, secrets and configMaps",
+		Short: "Render kubernetes controller manager bootstrap manifests, secrets and configMaps",
```
Comment (aveshagarwal) [inline cmd/cluster-kube-scheduler-operator/render/render.go]: kube-scheduler

Comment (sttts) [reply cmd/cluster-kube-scheduler-operator/render/render.go]: fixed

**Comment by aveshagarwal** [comment]
lets wait for tomorrow's demo before we get in, so that nothing breaks (what is already working) just in case.

**Pull request merged**